src/modules/views/Upload/useResumeFromFlagship.ts

```diff
@@ -5,14 +5,15 @@ import { useClient } from 'cozy-client'
 import { useWebviewIntent } from 'cozy-intent'
 import logger from 'cozy-logger'
 
-import { getUploadQueue, ADD_TO_UPLOAD_QUEUE } from 'drive/web/modules/upload'
-import { FileFromNative } from 'drive/web/modules/views/Upload/UploadTypes'
-import { getErrorMessage } from 'drive/web/modules/drive/helpers'
+import { getUploadQueue, ADD_TO_UPLOAD_QUEUE } from 'modules/upload'
+import { FileFromNative } from 'modules/views/Upload/UploadTypes'
+import { getErrorMessage } from 'modules/drive/helpers'
 
 export const useResumeUploadFromFlagship = (): void => {
   const client = useClient()
   const dispatch = useDispatch()
   const webviewIntent = useWebviewIntent()
+  // eslint-disable-next-line @typescript-eslint/no-unnecessary-type-assertion
   const uploadQueue = useSelector(getUploadQueue) as FileFromNative[]
```

Comment:
@acezard the typescript problem is here `getUploadQueue` is considered as any and the cast to FileFromNative as unnecessary. There are some other variable thats  considered as any in the file. Is this normal ?

Reply:
There was a problem with aliases when folders were moved out of the drive folder. I've added the main ones in the tsconfig to correct the problem.